geode-core/src/main/java/org/apache/geode/internal/cache/control/HeapMemoryMonitor.java

```diff
@@ -279,7 +279,9 @@ public void stopMonitoring() {
       NotificationEmitter emitter = (NotificationEmitter) ManagementFactory.getMemoryMXBean();
       try {
         emitter.removeNotificationListener(this, null, null);
-        this.cache.getLoggerI18n().fine("Removed Memory MXBean notification listener" + this);
+        if (logger.isTraceEnabled()) {
```

Comment:
I could be wrong but I thought "fine" mapped to "debug" not "trace". I think "trace" is for "finer" or "finest".
If I'm right then you should change all these new "trace" calls to "debug".